docs/termination.md

```diff
@@ -138,6 +140,51 @@ try {
 }
 ```
 
+### Inactivity timeout
+
+To terminate a subprocess when it becomes inactive, the [`cancelSignal`](#canceling) option can be combined with [transforms](transform.md) and some [debouncing logic](https://github.com/sindresorhus/debounce-fn). The following example terminates the subprocess if it has not printed to [`stdout`](api.md#resultstdout)/[`stderr`](api.md#resultstderr) in the last minute.
+
+```js
+import {execa} from 'execa';
+import debounceFn from 'debounce-fn';
+
+// 1 minute
+const wait = 60_000;
+
+const getInactivityOptions = () => {
+	const controller = new AbortController();
+	const cancelSignal = controller.signal;
+
+	// Delay and debounce `cancelSignal` each time `controller.abort()` is called
+	const abort = debounceFn(controller.abort.bind(controller), {wait});
+
+	const onOutput = {
+		* transform(data) {
+			// When anything is printed, debounce `controller.abort()`
+			abort();
```

Comment:
```suggestion
			scheduleAbort();
```

Just to make it clear it doesn't happen right away?

Reply:
Fixed in 493b3661855bd3b1c5f85dde2352682d25f28f8f